src/transformers/utils/quantization_config.py

```diff
@@ -271,3 +289,90 @@ def to_diff_dict(self) -> Dict[str, Any]:
                 serializable_config_dict[key] = value
 
         return serializable_config_dict
+
+
+@dataclass
+class AutoGPTQConfig(QuantizationConfig):
+    """
+    This is a wrapper class about all possible attributes and features that you can play with a model that has been
+    loaded using `optimum` api for gptq quantization relying on auto_gptq backend.
+    """
+
+    quant_method: QuantizationMethod = QuantizationMethod.GPTQ
```

Comment:
Any reason why we use `field` here? Can't we use the same structure as above (for consistency)?